src/components/hello/Hello.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
why you need `Hello.js`. Where is it used? If you don't need it, remove

Reply:
This used in Content.js accorded to require